src/worker/core/team/valueChange.ts

```diff
@@ -588,4 +624,53 @@ const valueChange = async (
 	return valuesAdd - valuesRemove;
 };
 
+const getModifiedPickRank = async (
+	tid: number,
+	pidsAdd: number[],
+	pidsRemove: number[],
+) => {
+	const teamSeason = await idb.cache.teamSeasons.indexGet(
+		"teamSeasonsBySeasonTid",
+		[tid, g.get("season")],
+	);
+	const teams = g.get("numActiveTeams");
```

Comment:
Rename variable to `numActiveTeams` to be more clear, `teams` usually is an array of teams.